SingularityExecutor/src/main/java/com/hubspot/singularity/executor/models/DockerEnvironmentContext.java

```diff
@@ -20,16 +21,12 @@ public DockerEnvironmentContext(TaskInfo taskInfo, List<String> inheritVariables
     List<Variable> variables = new ArrayList<>();
     Set<String> keys = new HashSet<>();
 
-    inheritVariables.forEach(
-      v -> {
-        if (!keys.contains(v)) {
-          String val = System.getenv(v);
-          if (val != null) {
-            variables.add(Variable.newBuilder().setName(v).setValue(val).build());
-            keys.add(v);
-          }
-        }
-      }
+    variables.add(
+      Variable
+        .newBuilder()
+        .setName("INHERIT_ENV_VARS")
```

Comment:
If we are going to loop over these, no need to make it another variable. Doing it this way will end up setting an extra variable that will be unused. Can add the list instead as another getter and loop over it in the template functions to avoid duplication/unused vars